integration/integration_test.go

```diff
@@ -0,0 +1,43 @@
+package integration_test
+
+import (
+	"compress/gzip"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func gunzipDB() string {
```

Comment:
Does it make sense to keep this method inside of `tar_input_test.go` as it's purely for testing and nothing else at the moment?

Reply:
I separated the file so I can add more tests later. As you said, it doesn't make sense for now. I can merge this function to `tar_input_test.go`.

Reply:
I addressed this. Moved it back to `tar_input_test.go` for now until we have the need for a separate file.